src/ripple/app/misc/LoadFeeTrack.h

```diff
@@ -42,46 +44,38 @@ class LoadFeeTrack
 {
 public:
     explicit LoadFeeTrack (beast::Journal journal = beast::Journal())
-        : m_journal (journal)
-        , mLocalTxnLoadFee (lftNormalFee)
-        , mRemoteTxnLoadFee (lftNormalFee)
-        , mClusterTxnLoadFee (lftNormalFee)
-        , raiseCount (0)
+        : j_ (journal)
+        , localTxnLoadFee_ (lftNormalFee)
+        , remoteTxnLoadFee_ (lftNormalFee)
+        , clusterTxnLoadFee_ (lftNormalFee)
+        , raiseCount_ (0)
     {
     }
 
-    virtual ~LoadFeeTrack () { }
-
-    // Scale from fee units to millionths of a ripple
-    std::uint64_t scaleFeeBase (std::uint64_t fee, std::uint64_t baseFee,
-        std::uint32_t referenceFeeUnits) const;
-
-    // Scale using load as well as base rate
-    std::uint64_t scaleFeeLoad (std::uint64_t fee, std::uint64_t baseFee,
-        std::uint32_t referenceFeeUnits, bool bUnlimited) const;
+    ~LoadFeeTrack() = default;
```

Comment:
Thanks for removing the un-needed `virtual`.  As a sanity check that no one was inheriting from `LoadFeeTrack` I decorated the class declaration (line 43) with `final` and rebuilt successfully.  You could consider leaving the class declaration adorned with `final` as a cheap compile-time check, but it's far from necessary.  Your call.